internal/storage/storage.go

```diff
@@ -153,14 +139,9 @@ func GetDevices(release_version diagnostics.ReleaseVersion) []Device {
 				Status:     DeviceStatus{ID: 1, Description: "healthy"},
 			}
 
-			if device.ID == mainDiskID {
-				device.MainDevice = true
```

Comment:
Will be always `false` not sure the implications

Reply:
It can be true, in case the `release_version` identified `mainDiskID` variable resulting of the call to `GetDeviceIdentifier` (removed in the PR) equals the disk identifier (for example, `/dev/sda`).
This is to identify the main disk but potentially as spoken via chat, can be greatly simplified :)